src/hyperfine/benchmark.rs

```diff
@@ -147,12 +147,12 @@ pub fn run_benchmark(
     shell_spawning_time: TimingResult,
     options: &HyperfineOptions,
 ) -> io::Result<()> {
-    println!(
-        "{}{}: {}",
-        "Benchmark #".bold(),
-        (num + 1).to_string().bold(),
-        cmd
-    );
+    let (benchark_text, current_num) = match options.output_style {
+        OutputStyleOption::Basic => ("Benchmark #".white(), (num + 1).to_string().white()),
```

Comment:
Unfortunately, `white()` will also output ANSI sequences (`[37mBenchmark #[0m[37m1[0m: sleep 0.1`). Could we simply keep the colored version here and disable coloring globally via [`colored::control::set_override`](https://docs.rs/colored/1.6.0/colored/control/fn.set_override.html)

Reply:
dang, I'm embarrassed I hadn't seen that option when I was looking at the docs for the crate. Thanks for pointing it out. I'll make the change back to the original for println statements and set the global output